src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/directionissued/DirectionIssuedAboutToStartHandler.java

```diff
@@ -88,4 +92,44 @@ private void setExtensionNextEventDropdown(State state, SscsCaseData sscsCaseDat
                 ? sscsCaseData.getExtensionNextEventDl().getValue() : new DynamicListItem("", "");
         sscsCaseData.setExtensionNextEventDl(new DynamicList(selectedValue, listOptions));
     }
+
+    private void clearFields(SscsCaseData sscsCaseData) {
+        sscsCaseData.setConfidentialityType(null);
+        sscsCaseData.setSendDirectionNoticeToFTA(null);
+        sscsCaseData.setSendDirectionNoticeToRepresentative(null);
+        sscsCaseData.setSendDirectionNoticeToOtherPartyRep(null);
+        sscsCaseData.setSendDirectionNoticeToOtherPartyAppointee(null);
+        sscsCaseData.setSendDirectionNoticeToOtherParty(null);
+        sscsCaseData.setSendDirectionNoticeToJointParty(null);
+        sscsCaseData.setSendDirectionNoticeToAppellantOrAppointee(null);
+    }
+
+    private void setPartiesToSendLetter(SscsCaseData sscsCaseData) {
+
+        YesNo hasOtherParty = isOtherPartyPresent(sscsCaseData) ? YesNo.YES : YesNo.NO;
+        YesNo hasOtherPartyRep = YesNo.NO;
+        YesNo hasOtherPartyAppointee = YesNo.NO;
+
+        if (YesNo.isYes(hasOtherParty)) {
+            boolean hasOtherPartyRepBoolean = sscsCaseData.getOtherParties().stream().map(CcdValue::getValue).anyMatch(OtherParty::hasRepresentative);
+            boolean hasOtherPartyAppointeeBoolean = sscsCaseData.getOtherParties().stream().map(CcdValue::getValue).anyMatch(OtherParty::hasAppointee);
+
+            if (hasOtherPartyRepBoolean) {
+                hasOtherPartyRep = YesNo.YES;
+            }
+
+            if (hasOtherPartyAppointeeBoolean) {
+                hasOtherPartyAppointee = YesNo.YES;
+            }
```

Comment:
Need test coverage for both of these if statements being false

https://sonarcloud.io/component_measures?metric=new_coverage&selected=SSCSTCA%3Asrc%2Fmain%2Fjava%2Fuk%2Fgov%2Fhmcts%2Freform%2Fsscs%2Fccd%2Fpresubmit%2Fdirectionissued%2FDirectionIssuedAboutToStartHandler.java&view=list&pullRequest=3222&id=SSCSTCA